src/URI.js

```diff
@@ -575,6 +575,10 @@
       string = '/' + string;
     }
 
+    if (parts.port) {
```

Comment:
this change should be covered by a test for the constructor (`test/test.js`, line 132)